### DISCUSSION TITLE
Missing galley_* metrics

### DISCUSSION THREAD

**Discussion by VonnyJap** [istio/istio#44643] [Q&A]
Can I get pointers on how galley metrics are logged into the `/metrics` endpoint? I am looking for metrics like `galley_validation_http_error`, according to the [docs](https://istio.io/v1.14/docs/reference/commands/pilot-discovery/#metrics), this should part of pilot-discovery monitoring, but when I scraped the metrics using Prometheus, I was not able to find it.

**Comment by howardjohn** [comment]
These metrics only show up when they are non-zero.

Simple reproducer:

```
$ curl -s localhost:15014/metrics | grep galley
# no match
$ curl https://localhost:15017/validate -k
no body found
$ curl -s localhost:15014/metrics | grep galley
# HELP galley_validation_http_error Resource validation http serve errors
# TYPE galley_validation_http_error counter
galley_validation_http_error{status="400"} 1
```
